### PR TITLE
Browser use with Llama 3.2 Vision Quickstart

### PR DESCRIPTION
**Pull request by miguelg719**
# Browser Use Llama-Recipe

This is an example notebook on how to create a Llama 3.2 vision-powered agent that can interact with web browsers on your behalf. It includes a detailed explanation of every section and example use cases. 

#### Features
- Visual understanding of web pages through screenshots
- Autonomous navigation and interaction
- Natural language instructions for web tasks
- Persistent browser session management

For example, you can ask the agent to:
- Search for a product on Amazon
- Find the cheapest flight to Tokyo
- Buy tickets for the next Warriors game

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/facebookresearch/llama-recipes/blob/main/CONTRIBUTING.md#pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes?  
- [ ] Did you write any new necessary tests?

Thanks for contributing 🎉!

### CODE REVIEW

File: recipes/use_cases/browser_use/agent/browser-use-quickstart.ipynb [hunk 1]
```diff
@@ -0,0 +1,683 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "few_shot_example_1 = \"\"\"\n",
+    "User Input: \"How much did Nvidia stock gain today?\"\n",
```
Comment (HamidShojanazeri) [inline recipes/use_cases/browser_use/agent/browser-use-quickstart.ipynb:361]: can we please change the query to a more neutral example.

File: recipes/use_cases/browser_use/agent/browser-use-quickstart.ipynb [hunk 2]
```diff
@@ -0,0 +1,683 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import base64\n",
+    "from IPython.display import Markdown\n",
+    "imagePath= \"screenshot.png\"\n",
```
Comment (HamidShojanazeri) [inline recipes/use_cases/browser_use/agent/browser-use-quickstart.ipynb:96]: do we need to add this screenshot in this folder as well? right now its been missing

**Comment by miguelg719** [comment]
Demo video added! @HamidShojanazeri

**Review by HamidShojanazeri — APPROVED** [review]

**Comment by HamidShojanazeri** [comment]
Thanks @miguelg719 great PR!

**Comment by aidando73** [comment]
That's cool

**Pull request merged**